website/content/en/preview/getting-started/getting-started-with-terraform/_index.md

```diff
@@ -361,11 +362,11 @@ resource "kubectl_manifest" "karpenter_provisioner" {
         cpu: 1000
     provider:
       subnetSelector:
-        karpenter.sh/discovery: ${local.cluster_name}
+        karpenter.sh/discovery: ${module.eks.cluster_id}
```

Comment:
`local.cluster_name` and `module.eks.cluster_id` are one in the same. Its better to use the module output though to ensure proper dependency ordering